changelog/8974.bugfix.md

```diff
@@ -0,0 +1,5 @@
+Intent IDs sent with events (to kafka and elsewhere) have been removed, intent
+names can be used instead (or if numerical values are needed for backwards
+compatibility, one can also hash the names to get previous ID values, ie.
+`hash(intent_name)` is the old ID values). Intent IDs have been removed because
+they are too big and can be turned to scientific notation causing confusion.
```

Comment:
"Intent IDs have been removed because as they were providing no extra value and integers that large were problematic for some event broker implementations." ?